package.json

```diff
@@ -29,7 +29,7 @@
     "chalk": "^1.1.3",
     "commander": "^2.9.0",
     "dotenv": "^6.0.0",
-    "handlebars": "4.5.2",
+    "handlebars": "4.7.6",
```

Comment:
Minor version increment (theoretically safe) + 80% coverage + All test pass + read the changelog = confident here